src/components/RenderHTML/BaseRenderHTML.js

```diff
@@ -192,20 +196,12 @@ const renderers = {
     edited: EditedRenderer,
 };
 
-const propTypes = {
-    /** HTML string to render */
-    html: PropTypes.string.isRequired,
-
-    /** Optional debug flag */
-    debug: PropTypes.bool,
-};
-
-const RenderHTML = ({html, debug = false}) => {
+const RenderHTML = ({html, debug = false, selectable = false}) => {
```

Comment:
It's redundant to have an inline default here in addition to using `defaultProps`. So just stick with `defaultProps`. Also, let's rename `selectable` to `textSelectable`:

```suggestion
const RenderHTML = ({html, debug, textSelectable}) => {
```

Reply:
ok